index.html

```diff
@@ -0,0 +1,214 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- google font -->
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+        href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap"
+        rel="stylesheet"
+    />
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+    <!-- fontawesome -->
+    <script src="https://kit.fontawesome.com/d31edfa6bf.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="./assets/css/main.css">
+    <link rel="stylesheet" href="./assets/css/common.css">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+        integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+    <title>Tech-Talk Conference</title>
+</head>
+<body>
+    <header id = "desk-most-top">
+        <ul class="most-top">
+            <li>
+                <a href="#" class="icon">
+                    <i class="fab fa-facebook fa-fw"></i>
+                </a>
+            </li>
+            <li>
+                <a href="#" class="icon">
+                    <i class="fab fa-twitter fa-fw"></i>
+                </a>
+            </li>
+            <li class="icon">English</li>
+            <li>
+                <a href="#" class="icon">My Page</a>
+            </li>
+            <li>
+                <a href="#" class="icon">Logout</a>
+            </li>
+        </ul>
+
+        <div id="main-nav">
+            <div>
+                <a href="#"  class="nav-logo"><img src="./assets/images/company_image.png" alt=""></a>
+            </div>
```

Comment:
- [ ] I noticed that all your images have empty `alt` values. Kindly use a valid string as `alt` value for all images on both pages. This would improve the accessibility of your application for visually impaired people.

Reply:
Noted and all solved